rust/connlib/tunnel/src/tests/dns_server_resource.rs

```diff
@@ -124,3 +98,24 @@ fn handle_dns_query(
 
     answers.into_message()
 }
+
+fn truncate_dns_response(message: &Message<Vec<u8>>) -> Vec<u8> {
```

Comment:
Why are we taking a reference here? Can't move it?

Reply:
I was just not changing the signature of the caller, `try_queue_udp_dns_response`

Reply:
in the original I mean, here we can move it, I just copy pasted